content/06.know-your-problem.md

```diff
@@ -1,20 +1,28 @@
-All of these different designs have different assumptions and caveats, which cannot be ignored during a data analysis.
-Many datasets are now passively collected or do not have a specific design, but even in this case it is important to know how individuals or samples were treated (for example, if all samples are from the same study site, if certain ethnic groups or zip codes are oversampled, if there are differences in processing dates or techniques.)
-
-Systematic biases can lead to artifacts or "batch effects," which mean that instead of finding correlates with an outcome or grouping of interest, the investigator may find correlates with variables that are not of interest and obtain misleading results [@doi:10.1038/nrg2825].
-Other study design considerations that should not be overlooked include knowing whether a study involves biological or technical replicates or both.
-For example, are some samples collected from the same individuals at different time points? Are those time points before and after some treatment?
-If one assumes that all the samples are independent but that is in fact not the case, a variety of issues may arise, including having a lower effective sample size than expected.
-
-In general, deep learning has an increased tendency for overfitting, compared to classical methods, due to the large number of parameters being estimated, making issues of adequate sample size even more important (see [Tip 7](#overfitting)).
-For a large dataset, overfitting may not be a concern, but the modeling power of deep learning may lead to more spurious correlations and thus incorrect interpretation of results (see [Tip 9](#interpretation)).
-Finally, it is important to note that with the exception of very specific cases of unsupervised data analysis, it is generally the case that a molecular or imaging dataset does not have much value without appropriate clinical or demographic data; this must always be balanced with the need to protect patient privacy (see [Tip 10](#privacy)). 
-Looking at these data can also clarify the study design (for example, by seeing if all the individuals are adolescents or women) or at least help the analyst employing deep learning to know what questions to ask.
+Once the data is obtained, it is easy to begin analyzing data without a good understanding of the study design, namely why the data were collected and how.
+Meta-data has been standardized in many fields and can help with this (for example, see [@doi:10.1038/ng1201-365]), but if at all possible, seek out a subject matter expert who has experience with this type of data.
+Receiving first-hand knowledge of the “gotchas" of a dataset will minimize the amount of guesswork and increase the success rate of the deep learning project.
+For example, if the main reason why the data were collected was to test the impact of an intervention, then it may be the case that a randomized controlled trial was performed.

```

Comment:
"data were" => "data was"

Reply:
No particular reason except consistency with some previous sentence. Some people prefer "data" as a plural-only (and refer to a measurement as "datum"), some people refer to data as singular. Personally, I usually use "data is" instead of "data are" but this is basically like a "VIM vs EMACS" argument :), but I really don't have a preference ... 

> Once the data is obtained, it is easy to begin analyzing data ...

We can also avoid the debate by writing

> Once the dataset is obtained, it is easy to begin analyzing the data ..."